style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: Roboto;
+}
+
+.calculator{
+  width: 248px;
+  height: 380px;
+  line-height: 1.5;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin: auto;
+  background-color: #adadad;
+  align-items: center;
+  border-radius: 10px;
+  box-shadow: 10px 10px 10px 5px grey;
+
+}
+
+
+.calculator #screen {
+  width: 248px;
+  height: 70px;
+  background-color: #adadad;
+  border-radius: 10px;
+  color: #f4f4f4;
+  text-align: right;
+  font-size: 60px;
+}
+.calculator .button{
```

Comment:
hey Jared, I've been working on my css and have been making unique naming conventions for most divs. However, I still feel the desire to sometimes use two classes. Along the lines of using .calculator-button for all the buttons AND .orange for the orange buttons because they keep most of the other css. Would you still recommend against this practice?